app/models/education_benefits_claim.rb

```diff
@@ -118,6 +118,10 @@ def selected_benefits
       benefits['vettec'] = true
     when '1990s'
       benefits['vrrap'] = true
+    when '1995'
+      benefit = parsed_form['benefit']&.underscore
+      benefits['chapter33'] = true if benefit.present? && benefit.start_with?('chapter33')
+      benefits[benefit] = true if benefit.present? && !benefit.start_with?('chapter33')
```

Comment:
Not a problem, but a nice shorthand for these lines could be:
```
      benefits['chapter33'] = true if benefit&.start_with?('chapter33')
      benefits[benefit] = true if !benefit&.start_with?('chapter33')
```

`benefit&.<method>` basically checks the presence of benefit before performing the next method in the chain. Should work for the negation as well!

Reply:
We'll be happy to make that adjustment on a subsequent push, but if you could approve asap as we have a frontend going in that this needs to be in synch with.